public/index.html

```diff
@@ -15,7 +15,5 @@
   </head>
   <body>
     <div id="app"></div>
-
-    <script data-jsd-embedded data-key="21430358-44e3-4787-8d72-e23b56714c03" data-base-url="https://jsd-widget.atlassian.com" src="https://jsd-widget.atlassian.com/assets/embed.js"></script>
```

Comment:
this is in master already right?

Reply:
Yeah, but Leon is working from that same branch because Git was acting up for him